packages/css/src/transformCss.ts

```diff
@@ -350,15 +354,49 @@ class Stylesheet {
         }
 
         this.transformSupports(root, mediaRule!['@supports'], conditions);
+        this.transformContainer(root, mediaRule!['@container'], conditions);
+      });
+    }
+  }
+
+  transformContainer(
+    root: CSSStyleBlock | CSSSelectorBlock,
+    rules: WithQueries<StyleWithSelectors>['@container'],
+    parentConditions: Array<string> = [],
+  ) {
+    if (rules) {
+      this.currConditionalRuleset?.addConditionPrecedence(
+        parentConditions,
+        Object.keys(rules).map((query) => `@container ${query}`),
+      );
+
+      forEach(rules, (containerRule, query) => {
+        const containerQuery = `@container ${query}`;
+
+        const conditions = [...parentConditions, containerQuery];
```

Comment:
Minor: Noticed that `containerQuery` is not used, as a hangover from `transformMedia`. But given this is now a carbon copy of `transformSupports` should we look at sharing code here? Or are we deliberately preferring verbosity for readability?

Reply:
I think I like these being separate even though they are basically the same. Not sure there's much to gain by merging them outside of less lines of code.